components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -128,6 +128,10 @@ public AuthenticatorFlowStatus process(HttpServletRequest request, HttpServletRe
                 return AuthenticatorFlowStatus.INCOMPLETE;
             }
         }
+
```

Comment:
Put a comment mentioning the reason for this method i.e the state parameter controls the flow.

Reply:
addressed with [85caf05](https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/144/commits/85caf05d600e67bcb8437cf354e6bff8867c872e)